recipes/nebari/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "nebari" %}
+{% set version = "2022.10.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nebari-{{ version }}.tar.gz
+  sha256: 02f59773c5b5035df53d156af3cf648bc88ce9384fc39ffa80291c8514d93980
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - nebari = nebari.cli.main:app
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - pip
+    - setuptools_scm
+  run:
+    - __unix
+    - python >=3.8,<4.0
+    - auth0-python
+    - bcrypt
+    - boto3
+    - cloudflare
+    - gitignore-parser ==0.0.8
+    - pydantic
+    - pynacl
+    - python-keycloak
+    - python-kubernetes
+    - ruamel.yaml
+    - azure-identity ==1.6.1
+    - azure-mgmt-containerservice ==16.2.0
+    - importlib_metadata
+    - rich
+    - requests-toolbelt <0.10.0,>=0.9.1
+    - typer
+    - questionary
+
+test:
+  imports:
+    - nebari
+    - nebari.cli
+  commands:
+    - pip check
+    - nebari --help
+    - if [[ $(nebari --version) != {{ version }} ]]; then exit 1; else exit 0; fi
```

Comment:
Nice! I'm wondering if it wouldn't make sense to include that in Grayskull, because `setuptools_scm`etc. is sometimes not included which this would catch.

Reply:
Good idea, that could be an extra custom setting that the Grayskull config could load -- e.g custom test class

Reply:
True, that works as well, but I just meant that Grayskull should add that to the recipes it generates so that the CI can catch this. :)